doc/monitoring/metrics_reference.rst

```diff
@@ -122,3 +139,76 @@ Requests:
 * ``tnt_net_requests_total`` - number of network requests this instance has handled since instance start
 
 * ``tnt_net_requests_current`` - amount of pending network requests
+
+.. _fibers:
+
+-------------------------------------------------------------------------------
+Fibers
+-------------------------------------------------------------------------------
+
+Provides statistics of fibers. If your app creates a lot of fibers, it can be used for monitoring
+fibers count and memory usage.
+
+* ``tnt_fiber_count`` - number of fibers
+
+* ``tnt_fiber_csw`` - averall fibers context switches number
+
+* ``tnt_fiber_memalloc`` - the amount of memory that is reserved for fibers
+
+* ``tnt_fiber_memused`` - the amount of memory that is used by fibers
+
+.. _operations:
+
+-------------------------------------------------------------------------------
+Operations
+-------------------------------------------------------------------------------
+
+Number of iproto requests this instance has processed, aggregated by request type.
+It can be used to find out which type of operation clients make more often.
+
+Two types of metrics provided for each request type:
+
+* ``tnt_stats_op_total`` - total number of calls since server start
+* ``tnt_stats_op_rps`` - number of calls for the last 5 seconds
+
+Each metric have ``operation`` label to be able to distinguish different request types, e.g.:
+``{operation="select"}``
+
+Request type could be one of:
+
+- ``delete`` - delete calls
+- ``error`` - requests resulted in an error
+- ``update`` - update calls
+- ``call`` - requests to execute stored procedures
+- ``auth`` - authentication requests
+- ``eval`` - calls to evaluate lua code
+- ``replace`` - replace call
+- ``execute`` - execute SQL calls
+- ``select`` - select calls
+- ``upsert`` - upsert calls
+- ``prepare`` - SQL prepare calls
+- ``insert`` - insert calls
+
+.. _replication:
+
+-------------------------------------------------------------------------------
+Replication
+-------------------------------------------------------------------------------
+
+Provides information of current replication status. To learn more about replication
+mechanism in Tarantool, see `this <https://www.tarantool.io/en/doc/2.6/book/replication/repl_architecture/#replication-mechanism>`_
```

Comment:
we should discuss with docs team how not to hardcode links to internal docs sections.